docs/utilities/tags.md

```diff
@@ -21,17 +21,45 @@ Using Tags In Code
 ------------------
 Block, Item, and Fluid tags are automatically sent from the server to any remote clients on login and reload. Function tags are not synced.
 
+Tags can be accessed in code through the `Tags.Items` and `Tags.Blocks` classes for all the Vanilla Minecraft tags, and through `ResourceLocation`s for ForgeTags.
+See the examples below for how to use each.
+
 `BlockTags.getCollection()` and `ItemTags.getCollection()` will retrieve the current `TagCollection`, from which you can retrieve a `Tag` object by its ID.
 With a `Tag` object in hand, membership can be tested with `tag.contains(thing)`, or all the objects in the tag queried with `tag.getAllElements()`.
 
-As an example:
+A reminder that `Tag`s contain `Item`s not `ItemStack`s. You can get the `Item` of an `ItemStack` from the stack by calling `<itemStack>.getItem()`.
+
+An example of checking if an Item belongs to a Vanilla `Tag`:
 ```Java
+//Retrieve whatever tag you want (using Tags.Blocks or Tags.Items)
+Tag<Item> tag = Tags.Items.DUSTS_REDSTONE;
+boolean inTag;
+// There's two methods to do this, doesn't really matter which you choose:
+// Method 1: from the Item's perspective
+inTag = someItemStack.getItem().isIn(tag);
+// Method 2: from the Tag's perspective
+inTag = tag.contains(someItemStack.getItem());
+```
+
+An example of checking if an Item is contained in a certain Forge `Tag`:
+```Java
+// Create a custom resource to use to fetch a tag instance ('mymod:myitemgroup' is the tag)
```

Comment:
This isn't generally correct, the tag should be a static constant somewhere, like all other tags.